docs/Formula-Cookbook.md

```diff
@@ -662,6 +648,9 @@ Formulae can specify an alternate download for the upstream project’s developm
 
 ```ruby
 class Foo < Formula
+  desc "Description"
```

Comment:
I think we could disable the description cop and remove all of these except e.g. `nginx`.

Reply:
I disabled the homepage cop too since that was similarly annoying.